pkg/k8s/apis/cilium.io/v2/register.go

```diff
@@ -34,7 +34,7 @@ const (
 	//
 	// Maintainers: Run ./Documentation/check-crd-compat-table.sh for each release
 	// Developers: Bump patch for each change in the CRD schema.
-	CustomResourceDefinitionSchemaVersion = "1.23.3"
+	CustomResourceDefinitionSchemaVersion = "1.24.0"
```

Comment:
@christarazi I already updated schema version, but it's not enough...?

Reply:
Ah, I somehow didn't notice it. Let's update it to 1.23.4 instead.